src/components/ForecastSummary.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "../styles/App.css";
+import "../styles/ForecastSummary.css";
+import iconData from "../data/iconData.json";
+
+function ForecastSummary(props) {
+  const { date, description, icon, temperature, onSelect } = props;
+  const formattedDate = new Date(date).toDateString();
+  const weatherCode = icon;
```

Comment:
The icons are not showing because the data is not set properly in the iconData.json, line 9 can be deleted and you could use it directly in the img element on line 16, src={iconData[icon]}